ajax/libs/react-block-ui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "react-block-ui",
+  "filename": "reactblockui.min.js",
+  "description": "Block UI for react",
+  "homepage": "https://availity.github.io/react-block-ui/",
+  "keywords": [
+    "availity",
+    "loader",
+    "blockui",
+    "redux",
+    "react",
+    "component",
+    "components",
+    "react-component",
+    "ui"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Availity/react-block-ui.git"
+  },
+  "license": "MIT",
+  "author": "Evan Sharp <evan.sharp@availity.com> (https://github.com/TheSharpieOne)",
+  "npmName": "react-block-ui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "reactblockui.min.js*",
+        "style.css"
```

Comment:
Yes, I suppose it wouldn't hurt to add the non-minified file. I will update the PR when I can. Thank you.